packages/eslint-config-react-app/index.js

```diff
@@ -315,5 +323,30 @@ module.exports = {
     'flowtype/define-flow-type': 'warn',
     'flowtype/require-valid-file-annotation': 'warn',
     'flowtype/use-flow-type': 'warn',
+
+    // https://github.com/testing-library/eslint-plugin-jest-dom
+    'jest-dom/prefer-checked': 'error',
+    'jest-dom/prefer-empty': 'error',
+    'jest-dom/prefer-enabled-disabled': 'error',
+    'jest-dom/prefer-focus': 'error',
+    'jest-dom/prefer-required': 'error',
+    'jest-dom/prefer-to-have-attribute': 'error',
+    'jest-dom/prefer-to-have-text-content': 'error',
```

Comment:
We could just use the recommended config instead

Reply:
We're always defining the rules ourselves, so that's why I went with it here too.